<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpriteGeneralDialog</class>
 <widget class="QDialog" name="SpriteGeneralDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0" rowspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="pictureWidthLabel">
       <property name="text">
        <string>原图宽：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextEdit" name="pictureWidthTextEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="pictureHeightLabel">
       <property name="text">
        <string>原图高：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="pictureHeightTextEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="hShardNumberLabel">
       <property name="text">
        <string>水平切分：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="hShardNumberSpinBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="vShardNumberLabel">
       <property name="text">
        <string>垂直切分：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="vShardNumberSpinBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="afterShardWidthLabel">
       <property name="text">
        <string>切分后宽：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QTextEdit" name="afterShardWidthTextEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="afterShardHeightLabel">
       <property name="text">
        <string>切分后高：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QTextEdit" name="afterShardHeightTextEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="pictureLabel">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="cancelButton1">
     <property name="text">
      <string>cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="spriteNameLabel">
       <property name="text">
        <string>sprite名字：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextEdit" name="spriteNameTextEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="totalActionNumberLabel">
       <property name="text">
        <string>总Action数：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="totalActionNumberSpinBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="spriteWidthLabel">
       <property name="text">
        <string>sprite宽：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spriteWidthSpinBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="spriteHeightLabel">
       <property name="text">
        <string>sprite高：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spriteHeightSpinBox"/>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>ok</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>pictureLabel</zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder>cancelButton1</zorder>
  <zorder>okButton</zorder>
  <zorder>verticalSpacer</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton1</sender>
   <signal>clicked(bool)</signal>
   <receiver>SpriteGeneralDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>673</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>SpriteGeneralDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
